Formula/wget.rb

```diff
@@ -23,16 +23,15 @@ class Wget < Formula
     depends_on "gettext"
   end
 
-  deprecated_option "enable-iri" => "with-iri"
   deprecated_option "enable-debug" => "with-debug"
 
-  option "with-iri", "Enable iri support"
   option "with-debug", "Build with debug support"
+  option "without-libidn", "Build without IRI support"
 
   depends_on "pkg-config" => :build
   depends_on "pod2man" => :build if MacOS.version <= :snow_leopard
-  depends_on "openssl" => :recommended
-  depends_on "libidn" if build.with? "iri"
+  depends_on "openssl"
+  depends_on "libidn" => :recommended
```

Comment:
What does `libidn` do? Not sure what IRI support is but I'd personally rather avoid the extra dependency given how infrequently this option is specified: 

<img width="664" alt="screen shot 2017-02-11 at 21 32 01" src="https://cloud.githubusercontent.com/assets/125011/22857681/9380aac6-f0a1-11e6-9133-f2d731950abd.png">